include/mqt-core/dd/Package.hpp

```diff
@@ -276,10 +279,12 @@ template <class Config> class Package {
       vectorInnerProduct.clear();
       vectorKronecker.clear();
       matrixKronecker.clear();
+      matrixTrace.clear();
       stochasticNoiseOperationCache.clear();
       densityAdd.clear();
       densityDensityMultiplication.clear();
       densityNoise.clear();
+      densityTrace.clear();
```

Comment:
Oh yes, indeed. It seems like this is also the case for some other tables, such as vectorKronecker and matrixKronecker, where only the final returned edge weight is stored in cUniqueTable. I will check the others as well!

Reply:
That could very well be the case. Some of these have only recently been refactored to just take a `Node` instead of an `Edge`.
Thanks for looking into it!

Reply:
As discussed offline, I transferred this to a separate issue. See #633